src/bundled.jsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import CioQuizComponent from './components/CioQuiz';
+import './styles.css';
+
+const CioQuiz = ({ selector, includeCSS = true, ...rest }) => {
+  if (document) {
+    const stylesheet = document.getElementById('cio-quizzes-styles');
+    const containerElement = document.querySelector(selector);
+
+    if (!containerElement) {
+      console.error(`ConstructorioQuizzes: There were no elements found for the provided selector`);
```

Comment:
Let's update this to `CioQuiz: There were....` since we're changing the window var